Tests/Fixtures/Configs/config-with-env-placeholder.yml

```diff
@@ -0,0 +1,18 @@
+output_dir: ${SWIFTGEN_OUTPUT_DIR}
+strings:
+  inputs: Sources1/Folder
+  outputs:
+    templateName: structured-swift3
+    output: strings.swift
+    params:
+      foo: 5
+      bar:
+        bar1: 1
+        bar2: 2
+        bar3:
```

Comment:
We should also check in the test (maybe a separate test) that env vars are NOT replaced where they are not supposed to. Especially if they are used as keys in the YAML they shouldn't be expanded, and similarly if they are not wrapped inside `${…}` either (so that `inputs: "Card$Game/Sources"` doesn't replace the `$Game` if there happen to be a `Game` env var defined